src/scripts/Build-Workbook.ps1

```diff
@@ -24,14 +24,59 @@ $Debug = $DebugPreference -eq "Continue"
 $outDir = Join-Path .. .. release "$Workbook-workbook"
 $workbookDir = Join-Path .. workbooks $Workbook
 
+# Define workbooks metadata
+$workbooksMetdata = @(
+    [PSCustomObject]@{
+        optimization = @{
+            "Product"     = "Azure Advisor"
+            "GalleryName" = "Cost Optimization"
+        }
+    }
+)
+
```

Comment:
We need it if we have other workbooks hosted under a [different product](https://github.com/microsoft/Application-Insights-Workbooks/tree/master/Workbooks) than Azure Advisor . If you think we won't have this we can remove it and hardcode the path

Reply:
See my other suggestion. You only need this because you're trying to match a file to the injection point. If you go the other way, you have everything you need already in the JSON. You can extract the folder name and inject the file based on that. Same basic concept, just more template-driven than folder-driven.